.github/workflows/fitbot.yml

```diff
@@ -51,7 +51,10 @@ jobs:
     - name: Installing NNPDF conda package
       shell: bash -l {0}
       run: |
-        conda create -n nnpdfenv -c file:///usr/share/miniconda/envs/test/conda-bld/linux-64/ nnpdf
+        NAME_VERSION=$(basename /usr/share/miniconda/envs/test/conda-bld/linux-64/nnpdf* .tar.bz2)
+        FINAL_NAME_VERSION=${NAME_VERSION#nnpdf-}
+        FINAL_NAME_VERSION=${FINAL_NAME_VERSION%-py*}
+        conda create -n nnpdfenv -c file:///usr/share/miniconda/envs/test/conda-bld/linux-64/ -c defaults -c conda-forge  nnpdf=$FINAL_NAME_VERSION
```

Comment:
What does al this do?

Reply:
Let’s leave that for the end, but yes, it should eventually be reverted.